dashboard/src/t5gweb/ui.py

```diff
@@ -202,12 +202,12 @@ def login():
 @login_required
 def index():
     """list new cases"""
-    load_data()
+    plot_data = plots()         
     return render_template(
         "ui/index.html",
-        new_cases=load_data.new_cases,
-        values=load_data.y,
-        now=load_data.now,
+        new_cases=get_new_cases(),
+        values=list(plot_data.values()),
+        now=redis_get("timestamp"),
```

Comment:
as per issue #274 we are looking to remove the redis_get/set calls wherever possible. could you add a function to utils.py, say `get_timestamp` to wrap `redis_get("timestamp")` and call that instead?

@adhil0 is that in line with your thinking on #274?

Reply:
My thinking is kind of the opposite, I think. Basically to break redis_get() calls out of functions, rather than wrapping them. So instead of 
```python
def get_new_comments():
    redis_get("cards")
...

get_new_comments()
```
It would be
```python
def get_new_comments(cards):
...

cards = redis_get("cards")
get_new_comments(cards)
```
This way its easier to write tests for functions, and we can call the cache once and use the same data across multiple functions, rather than calling the cache individually in each function. So I think the way it is in this PR is OK IMO

Reply:
yeah, LTGM @adhil0